### PR TITLE
[OPMONDEV-161]: create opendata_collector_module and parse settings

### PR DESCRIPTION
**Pull request by wisecrow**
https://nordic-institute.atlassian.net/browse/OPMONDEV-161

### CODE REVIEW

File: opendata_collector_module/metrics_opendata_collector/opendata_collector.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#  THE SOFTWARE.
+
+
+def collect_opendata(settings, source_settings):
```
Comment (justinask7) [inline opendata_collector_module/metrics_opendata_collector/opendata_collector.py:24]: type hints would be nice

File: opendata_collector_module/metrics_opendata_collector/settings.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+from os import listdir
+from os.path import isfile, join
+from typing import List, Optional
+
+import yaml
+
+
+class MetricsSettingsManager:
+    """
+    Class to hold Metrics settings.
```
Comment (justinask7) [inline opendata_collector_module/metrics_opendata_collector/settings.py:33]:
```suggestion
    """ Class to hold Metrics settings.
```

File: opendata_collector_module/setup.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      description='X-Road Metrics Opendata Collector Module',
+      long_description='',
+      author='NIIS',
+      author_email='info@niis.org',
+      packages=['metrics_opendata_collector'],
+      install_requires=requirements,
+      classifiers=classifiers,
+      platforms='POSIX',
+      license='MIT'
+      )
```
Comment (justinask7) [inline opendata_collector_module/setup.py:58]:
```suggestion
)
```

**Review by justinask7 — APPROVED** [review]
Lgtm 🙌

File: opendata_collector_module/tox.ini [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+[testenv:py38-mypy]
+description = Run mypy
+deps =
+    mypy
+    {[testenv]deps}
+commands =
+    mypy --install-types --non-interactive {toxinidir}/metrics_opendata_collector/
+
+[pycodestyle]
+max-line-length = 160
```
Comment (justinask7) [inline opendata_collector_module/tox.ini:21]: 160 seems quite a lot, maybe 120 would make more sense? Small point, but still a point

File: opendata_collector_module/metrics_opendata_collector/settings.py (hunk 2 above)
Comment (wisecrow) [reply opendata_collector_module/metrics_opendata_collector/settings.py]: Fixed.

File: opendata_collector_module/metrics_opendata_collector/opendata_collector.py (hunk 1 above)
Comment (wisecrow) [reply opendata_collector_module/metrics_opendata_collector/opendata_collector.py]: Added.

**Pull request merged**